eng/pipelines/official.yml

```diff
@@ -251,6 +251,8 @@ stages:
     variables:
       # Gets the PackageVersion variable produced by the Build pipeline.
       PackageVersion: $[ stageDependencies.Build.BuildOfficialRelease.outputs['SetPackageVersion.PackageVersion'] ]
+      # Gets the AssemblyVersion variable produced by the Build pipeline.
+      AssemblyVersion: $[ stageDependencies.Build.BuildOfficialRelease.outputs['SetAssemblyVersion.AssemblyVersion'] ]
```

Comment:
Just so I'm sure I understand how this works, `SetAssemblyVersion` here refers to the name of the task that set the `AssemblyVersion` variable?

Reply:
Correct. The syntax actually becomes more confusing based on the scenario (meaning, where and how the variable needs to be accessed). But, this syntax is what works in `stage` variables.